src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Level,
+  LevelItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+  Title,
+} from "@patternfly/react-core";
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type SettingsProps = {
+  showAlert: AlertProps[];
+  isChecked: boolean;
+  isModalOpen: boolean;
+  onChange: () => void;
+  onDisable: () => void;
+  onClose: () => void;
+  closeAlertAction: (key: string | undefined) => void;
+  isLoading: boolean;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  showAlert,
+  isChecked,
+  onChange,
+  isModalOpen,
+  onDisable,
+  onClose,
+  closeAlertAction,
+  isLoading,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection isWidthLimited isCenterAligned>
+          <Card>
+            <CardBody>
+              <Level>
+                <LevelItem>
+                  <Flex>
+                    <FlexItem spacer={{ default: "spacerNone" }}>
+                      <Title headingLevel={"h3"}>
```

Comment:
Ideally, the text that represents the setting label and value would be inline elements grouped together within the same block-level element so that the text is treated as one block of text (i.e. read aloud as one block of text by screen readers). This helps screen reader users easily know what the current value of the setting is when they're reading this text.

For example:
`<strong>Connection re-authentication:</strong> On`

The `<strong>` element here provides emphasis on the label. This provides both visual separation between the label and value for sighted users, but also this emphasis helps differentiate these strings when announced by a screen reader. 

To provide this text as a single block, you can make these changes:
- On the `<FlexItem>` add `component="span"` to make these inline elements instead of block elements.
- Replace the `<Title>` component with a `<strong>` element. The Title component results in a block level element, whereas `<strong>` is an inline element.